test/compflags/checks/checkModByZeroScope.good

```diff
@@ -1 +1 @@
-checkDivByZero.chpl:7: error: halt reached - Attempt to divide by zero
+checkModByZeroScope.chpl:7: error: halt reached - Attempt to compute a modulus by zero
```

Comment:
This test passes with LLVM, but still hits the failure in the future with a gcc backend.  I took a shot at this in #24133.